src/commands/github.ts

```diff
@@ -14,6 +14,28 @@ const command: Command = {
 			name: "name",
 			description: "The type of item to fetch (branch, commit, issue, pullrequest)",
 			required: true,
+			choices: [
+				{
+					name: "Branch",
+					value: "branch",
+				},
+				{
+					name: "Commit",
+					value: "commit",
+				},
+				{
+					name: "Issue",
+					value: "issue",
+				},
+				{
+					name: "Pull Request",
+					value: "pull",
```

Comment:
The switch statement below does not account for "pull", however there is no reason to account for different variations of "pull" (e.g., "pull-request", "pullrequest", etc.) now that we have choices.